plugins/woocommerce/src/Internal/Admin/EmailPreview.php

```diff
@@ -85,6 +85,13 @@ private function render_legacy_preview_email() {
 	 * @return string
 	 */
 	private function render_preview_email() {
+		/**
+		 * Always show shipping address in the preview email.
+		 *
+		 * @since 9.5.0
+		 */
+		add_filter( 'woocommerce_order_needs_shipping_address', '__return_true' );
```

Comment:
Should we remove this filter after the preview email content is generated? E.g. after [this line](https://github.com/woocommerce/woocommerce/pull/52415/commits/cafd1730b626f56886a8246ff14cb92ab7e9bfa8#diff-837c7a5181d152fc6cb6a7a2673e3cd7ecdb87bb68989f0e01aca341622b6751L93)? Or is it unnecessary and won't interfere with any subsequent user hooks like `woocommerce_mail_content`?

Reply:
Good point. Not sure if it would really interfere, but it doesn't harm adding it. Added it cb2bd1c (also for `woocommerce_order_item_product`).